### PR TITLE
Test for code-snippet

### PR DESCRIPTION
**Pull request by inuscript**
Example code has mistake sometime.
I think code snippet example maybe need testing.

Test code and structure is draft ( only `compact, concat, fill`) .

### CODE REVIEW

File: tests/unit/all.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  assert.deepEqual(
+    _.fill([4, 6, 8, 10], '*', 1, 3),
+    [4, 6, 8, 10].fill('*', 1, 3)
+  )
+}
+
+// Run tests
+Object.entries(tests).forEach(([name, testFn]) => {
+  testFn()
+})
```
Comment (inuscript) [inline tests/unit/all.js]: * I create simple testing.
* may use testing framework ( like mocha ) and split test command ( like `npm run test:unit`)

File: tests/index.js [hunk 2]
```diff
@@ -11,3 +11,6 @@ assert.equal(plugin.configs['compatible-warn'].rules['you-dont-need-lodash-under
 assert.equal(plugin.configs['compatible-warn'].rules['you-dont-need-lodash-underscore/last-index-of'], 1);
 assert.equal(plugin.configs.compatible.rules['you-dont-need-lodash-underscore/for-each'], 1);
 assert.equal(plugin.configs.compatible.rules['you-dont-need-lodash-underscore/is-nan'], 2);
+
+// snippet test
+require('./unit/all.js');
```
Comment (inuscript) [inline tests/index.js]: I append test run entrypoint to exist testing.
But it's may use testing framework ( like mocha ) and split test command ( like npm run test:unit)

**Comment by cht8687** [comment]
Good idea~

**Comment by inuscript** [comment]
@stevemao May I merge it? (approve?)

**Pull request merged**